tensorflow-core/tensorflow-core-api/src/main/java/org/tensorflow/Graph.java

```diff
@@ -125,6 +125,66 @@ public GraphOperation operation(String name) {
     }
   }
 
+
+  /**
+   * Returns the operation (node in the Graph) with the provided name.
+   * <p>
+   * Or throws an {@code IllegalArgumentException} if no such operation exists in the Graph.
+   *
+   * @param name name of the operation to look for
+   * @return operation in the graph with this name
+   * @see #operation(String)
+   */
+  public GraphOperation operationOrError(String name) {
```

Comment:
Kind of, eventually I'd want to mark it `@NonNull`.  I moved the method here from session, where it did throw like that rather than returning null.  Since we already had the `operation` in Graph I added this as `operationOrError` rather than moving the check to any uses.